### PR TITLE
Print Trino version to EXPLAIN ANALYZE VERBOSE

### PR DESCRIPTION
**Pull request by sopel39**
Output of EXPLAIN ANALYZE VERBOSE is often
shared for troubleshooting therefore it's
useful to print Trino version there.

** NO RELEASE NOTES **

### CODE REVIEW

File: core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java [hunk 1]
```diff
@@ -418,6 +422,10 @@ public static String textDistributedPlan(
                 .collect(toImmutableMap(DynamicFilterDomainStats::getDynamicFilterId, identity()));
         TypeProvider typeProvider = getTypeProvider(allFragments);
 
+        if (verbose) {
```
Comment (raunaqmorarka) [inline core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java:425]: Why not print this always ?

**Review by raunaqmorarka — COMMENTED** [review]
Please update the examples in `explain.rst` and `explain-analyse.rst`

File: core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java (hunk 1 above)
Comment (sopel39) [reply core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java:425]: I would say `EXPLAIN ANALYZE` is more for end-users while `EXPLAIN ANALYZE VERBOSE` it's kind of diagnostic output for devs and power users. Therefore, I thought it's not that useful to keep it in plain `EXPLAIN ANALYZE`

**Review by raunaqmorarka — APPROVED** [review]

File: core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java (hunk 1 above)
Comment (findepi) [reply core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java:425]: Why not always?

every EXPLAIN is quite verbose, and version string is quite short, so having it in every EXPLAIN & EXPLAIN ANALYZE wouldn't hurt

**Pull request merged**